lms/static/coffee/src/navigation.js

```diff
@@ -0,0 +1,70 @@
+var edx = edx || {},
```

Comment:
It would be good to move this out of the ``coffee`` directory as this isn't a coffee file anymore. I recommend putting it in ``lms/static/js/components` if it a purely LMS feature. If this is a pattern library component, I'd probably put it in ``common/static/common/js/components``.